src/main/scala/beam/agentsim/agents/household/HouseholdActor.scala

```diff
@@ -189,6 +194,8 @@ object HouseholdActor {
     private var cavPassengerSchedules: Map[BeamVehicle, PassengerSchedule] = Map()
     private var personAndActivityToCav: Map[(Id[Person], Activity), BeamVehicle] = Map()
     private var personAndActivityToLegs: Map[(Id[Person], Activity), List[BeamLeg]] = Map()
+    private var householdMembersToLocationTypeAndLocation: Map[Id[Person], (ParkingActivityType, String, Coord)] = Map()
```

Comment:
I realize there are some cases where this has been done already, but we really should be using a `mutable.Map` instead of `var`. https://beam.readthedocs.io/en/latest/developers.html#use-mutable-buffer-instead-of-immutable-var